src/common/telemetry/src/logging.rs

```diff
@@ -242,5 +265,8 @@ pub fn init_global_logging(
             .expect("error setting global tracing subscriber");
     }
 
+    // report metric base on config remote_write
+    report_metric_task(opts.remote_write.clone());
```

Comment:
Don't do this here. This function is only for setting up logging layers. A better place is letting each server (standalone/frontend etc.) start their own reporting task, as this might require a network connection.

And this config should be put on the server side instead of being part of telemetry options